### DISCUSSION TITLE
为了这个官方的集成特意买了中枢网关，遇到两个问题。

### DISCUSSION THREAD

**Discussion by PerryLayne** [XiaoMi/ha_xiaomi_home#252] [Q&A]
README_zh.md中说：

> 米家集成是否支持本地化控制？

米家集成支持通过[小米中枢网关](https://www.mi.com/shop/buy/detail?product_id=15755&cfrom=search)（固件版本 3.4.0_000 以上）

1.但我入手后，更新中枢网关到最新版发现最新版是3.3.0，这是什么情况是现在中枢不支持吗？
2.在HA中哪里能看到是否已经启用中枢的本地模式呢？我并没有找到任何提示，难道我中枢白买了吗

**Comment by Hugh696** [comment]
等中枢固件更新，别着急

**Reply by NOisi-x** [reply]
往下读一读，新版固件还没发布呢

**Comment by summershrimp** [comment]
三月份了，还不推送。。

**Reply by Hugh696** [reply]
过年的时候就推送了，你没收到嘛

**Reply by zy900906** [reply]
年前是收到了,但不是3.4,但是描述上又好似是本地化,但我目前测试,并没有本地化,把网断了,自动化就瘫痪了,我用的是HA+NR

**Reply by haoxian2023** [reply]
> 过年的时候就推送了，你没收到嘛

是会有很多人在看到up主去年10月份已经宣传0.8.7的版本，而在接近半年后的今天还没收到更新的，比如我😂😂

**Reply by summershrimp** [reply]
没收到啊。。已退回miot_auto插件。。发现用那个插件绝大部分东西都可以通过wifi直连或中枢网关控制了

**Reply by PerryLayne** [reply]
> 没收到啊。。已退回miot_auto插件。。发现用那个插件绝大部分东西都可以通过wifi直连或中枢网关控制了

啊？？这个插件支持中枢了吗？

**Reply by summershrimp** [reply]
> > 没收到啊。。已退回miot_auto插件。。发现用那个插件绝大部分东西都可以通过wifi直连或中枢网关控制了
> 
> 啊？？这个插件支持中枢了吗？

也可能是通过多模控制的蓝牙设备? 反正这个插件我选仅本地的模式几乎没看到没加入成功的设备

**Comment by rumianoesa** [comment]
我也是买了才发现根本没有3.4的固件

**Comment by summershrimp** [comment]
终于收到了 0.8.9

**Reply by CODPHISH** [reply]
小米中枢网关更新了吗？我才买
